cardano-sync/src/Cardano/Sync/Types.hs

```diff
@@ -66,3 +67,14 @@ newtype BlockId = BlockId Int
 -- @Word64@ is valid as well.
 newtype MetaId = MetaId Int
     deriving (Eq, Show)
+
+-- The metrics we use.
+-- PLEASE do not put this into ENV, it's defeating the purpose
+-- of these constructs.
```

Comment:
That is not a particularly useful comment.  It tells me do not do something, but now why I should not do it.

How about "Kept as a separate struct because ......."

Reply:
Will update.